src/orange/universes/SimpleUnitTracker.hh

```diff
@@ -224,15 +287,20 @@ SimpleUnitTracker::intersect_impl(const LocalState& state, F is_valid) const
         // no "nearby" distances depending on F)
         return {};
     }
-    else if (is_simple)
+    else if (vol.flags() == 0)
     {
-        // No interior surfaces: closest distance is next boundary
+        // No special conditions: closest distance is next boundary
         return this->simple_intersect(state, vol, num_isect);
     }
-    else
+    else if (!is_simple)
```

Comment:
Why not just have this be the `else` clause?

Reply:
Yeah, I changed this while I was adding "background cell" tracking (which needs another flag and tests another conditional here)—I can revert for the time being.